doc/dev/documenting.rst

```diff
@@ -2,6 +2,20 @@
  Documenting Ceph
 ==================
 
+User documentation
+==================
+
+The documentation on docs.ceph.com is generated from the restructuredText
+sources in ``/doc/`` in the Ceph git repository.
```

Comment:
github view should now render the rst doc properly? there is also restview that one can install on local host: https://pypi.python.org/pypi/restview